schemas/2015-08-01/Microsoft.ServiceBus.json

```diff
@@ -0,0 +1,1432 @@
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2015-08-01"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Subscription data center location."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/SubscriptionProperties"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.ServiceBus/namespaces/topics/subscriptions"
+    }
+  },
+  "definitions": {
+    "MessageCountDetails": {
+      "type": "object",
+      "properties": {
+        "activeMessageCount": {
+          "oneOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Number of active messages in the queue, topic, or subscription."
+        },
+        "deadLetterMessageCount": {
+          "oneOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Number of messages that are dead letters."
+        },
+        "scheduledMessageCount": {
+          "oneOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Number scheduled messages."
+        },
+        "transferDeadLetterMessageCount": {
+          "oneOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Number of messages transferred into dead letters."
+        },
+        "transferMessageCount": {
+          "oneOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Number of messages transferred to another queue, topic, or subscription."
+        }
+      },
+      "description": "Message Count Details."
+    },
+    "NamespaceProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```

Comment:
this looks like a read-only property, does your service expect it in PUT request body?